packages/docs-app/src/examples/datetime-examples/dateInputExample.tsx

```diff
@@ -44,9 +44,11 @@ export class DateInputExample extends BaseExample<IDateInputExampleState> {
     );
 
     protected renderExample() {
+        const { formatKey, ...otherState } = this.state;
         return (
             <DateInput
-                {...this.state}
+                format={FORMATS[formatKey]}
+                {...otherState}
```

Comment:
The position of the spread in the prop ordering suggests nuance when none exists. Can we just do:

```tsx
<DateInput
    {...spreadableState} {/* maybe a better name? */}
    format={FORMATS[formatKey]}
    ...
>
```